client/chain/chain.go

```diff
@@ -342,6 +347,29 @@ type chainClient struct {
 	canSign bool
 }
 
+func (cc *chainClient) loadOfacList() error {
+	response, err := http.Get(defaultOfacListURL)
+	if err != nil {
+		return err
+	}
+	defer response.Body.Close()
+
+	if response.StatusCode != http.StatusOK {
+		return fmt.Errorf("request to the OFAC upstream failed with code: %s", response.Status)
+	}
+
+	body, err := io.ReadAll(response.Body)
+	if err != nil {
+		return err
+	}
+
+	var ofacList []string
+	if err := json.Unmarshal(body, &ofacList); err != nil {
+		return err
+	}
+	return nil
+}
```

Comment:
As per in the Python SDK, I think the idea is to keep a local copy of the ofac.json file. The helper method to download the file will be usefull (and I think required), but it should be in a separate module.
The ChainClient should just load the addresses from the local file